airflow-core/src/airflow/ui/src/i18n/locales/pl/common.json

```
@@ -100,17 +108,26 @@
   "switchToDarkMode": "Przełącz na tryb ciemny",
   "switchToLightMode": "Przełącz na tryb jasny",
   "table": {
+    "completedAt": "Zakończono o",
+    "createdAt": "Utworzono o",
+    "duration": "Czas trwania",
     "filterByTag": "Filtruj Dagi według tagu",
     "filterColumns": "Filtruj kolumny tabeli",
     "filterReset_one": "Resetuj filtr",
     "filterReset_other": "Resetuj filtry",
+    "from": "Od",
+    "maxActiveRuns": "Maksymalna liczba aktywnych uruchomień",
```

```suggestion
    "maxActiveRuns": "Maksymalna liczba aktywnych wykonań",
```
I think this will be consistent with other translations where we translate `run` as `wykonanie`.

---

Indeed - wykonanie is better in general.